### ISSUE TITLE
Support --activity-profile-out and --assembly-region-out in HaplotypeCallerSpark

### ISSUE THREAD

**Issue by tomwhite** [broadinstitute/gatk#5503]

**Comment by jamesemery** [comment]
Of the outputs to HaplotypeCaller you have created issues for, my understanding is that this one in particular is mostly a debug tool. It may be helpful in our efforts to improve the spark version to implement this feature but its probably not a high priority.

**Comment by tomwhite** [comment]
Agreed. Also, given that it's difficult to get Spark to write to multiple output types at once, it would probably be easiest for these options to take the place of the final output, which would probably be OK since it's for debugging. Alternatively, there could be another tool for producing these outputs.
